docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM php:7.3-cli-alpine3.12
```

Comment:
Yes it's only required for local development. Basically I did it so I don't have to install any PHP extensions on my host but can spin up a test environment within a few minutes where I can run the unit tests (which need at least xdebug or pcov).

I used PHP 7.3 because it's the minimum requirement of this package (https://github.com/friends-of-phpspec/phpspec-code-coverage/blob/master/composer.json#L41). Of course we need to test it against many other versions but at least I immediately realize during development whether I use a feature that is not available in PHP 7.3 and if the code works on the minimum required PHP version.